src/daemon/daemon.cpp

```diff
@@ -2916,6 +2916,22 @@ void mp::Daemon::create_vm(const CreateRequest* request,
     // TODO: We should only need to query the Blueprint Provider once for all info, so this (and timeout below) will
     //       need a refactoring to do so.
     const std::string blueprint_name = config->blueprint_provider->name_from_blueprint(request->image());
+    // if the launch target is a blueprint instead of an image, issues a deprecation warning
+    if (!blueprint_name.empty())
+    {
+        constexpr auto deprecation_warning =
+            "**Warning! Blueprints are deprecated and will be removed in a future release.**\n\n"
```

Comment:
```suggestion
            "*** Warning! Blueprints are deprecated and will be removed in a future release. ***\n\n"
```

To stick with the format we used for hyperkit.